pkg/gds/admin_test.go

```diff
@@ -410,7 +411,7 @@ func (s *gdsTestSuite) TestSummary() {
 		VASPsCount:           14,
 		PendingRegistrations: 6,
 		ContactsCount:        39,
-		VerifiedContacts:     26,
+		VerifiedContacts:     24,
```

Comment:
I think this is the only remaining mystery?

Reply:
I think I've cracked this, there are some tests which modify the fixtures in the reference map. To get around this we usually defer `LoadReferenceFixtures` to reload the references for the following tests. This was missing in `RetrieveVASP` because `CompareFixture` nils out some of the VASP fields before comparison. We had also previously put in some deep copy code into `CompareFixture` to avoid modifying the values in the reference fixtures. However, I think the more user friendly option is just to call `LoadReferenceFixtures` in the 3-4 tests that need to modify the reference objects so I've removed the deep copy code as it was a bit hard to follow.